### ISSUE TITLE
multiple work places

### ISSUE THREAD

**Issue by mathisdt** [mathisdt/trackworktime#3]
support multiple work places for location-based tracking (latitude, longitude, tolerance)

**Comment by HerHde** [comment]
You're not talking about multiple employers with their own time calculations here, right?

**Comment by mathisdt** [comment]
That's right, in this issue it's only planned to have multiple work locations for the same employer. The time clocked at any of the locations should count for this one employer.
